Enable/Utilities/ErrorHandler.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ErrorHandler.h
+//  Enable
+//
+//  Created by Amanda Wang on 7/12/22.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+NS_ASSUME_NONNULL_BEGIN
+//TODO: handle errors with network connection https://stackoverflow.com/questions/1083701/how-can-i-check-for-an-active-internet-connection-on-ios-or-macos
```

Comment:
I assume this is covered, but just to make sure we're on the same page. In one hand you can check network availability and that would give you a quick way to say "There's on internet connection, retry, etc.". The second part of it is that every network request could fail (e.g. http 500, 404, etc.) and those also have to be handled accordingly.